tests/fixtures/names.fixture.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  GeneralRoman_Cs,
+  GeneralRoman_En,
+  GeneralRoman_EnGb,
+  GeneralRoman_EnIe,
+  GeneralRoman_Es,
+  GeneralRoman_Fr,
+  GeneralRoman_It,
+  GeneralRoman_La,
+  GeneralRoman_Pl,
+  GeneralRoman_PtBr,
+  GeneralRoman_Sk,
+} from 'romcal/dist/bundles/general-roman';
+
+export const fixture = [
```

Comment:
I think it might be benefitial to add a type of this fixture. Note that I have no idea what type is 

```suggestion
import { BaseRomcalBundle } from '../lib/types/bundle'

type Fixture = ({
  c: BaseRomcalBundle,
  t: ({
    k: string,
    v: string,
  })[]
})[]

export const fixture = [
```

Reply:
i'm down for this, sorry I didn't see it till after you merged!

Reply:
Sorry for merging it before you had a chance to look at it.

> i'm down for this

You mean you are against it? Why then?

I am even thinking to move the `type` definition to a separate file (like into `tests/type/fixture.type.ts`, as it is only used in tests). I want to import it in another test I have just finished for a bug in Latin.

Reply:
no, sorry, another American colloquialism. I think this is a good thing.

Reply:
Yeah, meanwhile I’ve looked it up. I was under impression that `down for sth` is used when you are against it and `up for sth` when you are for something, however, now I know both are used positively (strangely IMO :wink:). At least I learn something new—keep using the phrases you are used to! :pray:

Reply:
i don't think i've ever heard down for used negatively, haha.

Reply:
I didn’t say I’ve heard it used that way either, however, I _thought_ it is used that way. :wink: